brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-856 Added hover box shadow option for container block

### Description
In the Box Shadow section, there’s no way to add a different box shadow for hover. I think we should consider adding a hover option here. This will bring more design flexibility to our users.

### Screenshots
[](https://share.getcloudapp.com/Z4uAxDYv)

### Types of changes


Improvement


### How has this been tested?
- Apply normal and hover box shadow for a container
- Check in the editor and front end too

### Checklist:
